scm/driver/stash/content.go

```diff
@@ -53,6 +68,13 @@ type contents struct {
 	Values []string `json:"values"`
 }
 
+type contentCreateUpdate struct {
```

Comment:
the current design in scm service is that whenever we are calling the client.do method, we pass it the json value and the client.do method converts it into the multipart data.
Because of which we cannot name it to multipartContent